distributed/deploy/tests/test_local.py

```diff
@@ -1155,3 +1155,22 @@ async def test_connect_to_closed_cluster():
         # Raises during init without actually connecting since we're not
         # awaiting anything
         Client(cluster, asynchronous=True)
+
+
+class MyPlugin:
+    def setup(self, worker=None):
+        import my_nonexistent_library  # noqa
+
+
+@gen_test(
+    clean_kwargs={
+        # FIXME: This doesn't close the LoopRunner properly, leaving a thread around
```

Comment:
@graingert this is leaving the thread of a loop runner alive. I didn't want to mess with this but it may interest you